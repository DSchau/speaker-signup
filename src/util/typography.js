import Typography from 'typography';

export default new Typography({
  googleFonts: [
    {
      name: 'Lato',
      styles: ['400', '900'],
    },
  ],
});
