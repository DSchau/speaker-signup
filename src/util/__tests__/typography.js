import typography from '../typography';

test('it is the default export', () => {
  expect(typography).toBeDefined();
});

test('it matches snapshot', () => {
  expect(typography.toJSON()).toMatchSnapshot();
});
