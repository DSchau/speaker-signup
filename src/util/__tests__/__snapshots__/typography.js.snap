// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it matches snapshot 1`] = `
Object {
  "*": Object {
    "boxSizing": "inherit",
  },
  "*:after": Object {
    "boxSizing": "inherit",
  },
  "*:before": Object {
    "boxSizing": "inherit",
  },
  "abbr": Object {
    "borderBottom": "1px dotted hsla(0,0%,0%,0.5)",
    "cursor": "help",
  },
  "abbr[title]": Object {
    "borderBottom": "1px dotted hsla(0,0%,0%,0.5)",
    "cursor": "help",
    "textDecoration": "none",
  },
  "acronym": Object {
    "borderBottom": "1px dotted hsla(0,0%,0%,0.5)",
    "cursor": "help",
  },
  "address": Object {
    "marginBottom": "1.61rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "b": Object {
    "fontWeight": 700,
  },
  "blockquote": Object {
    "marginBottom": "1.61rem",
    "marginLeft": "1.61rem",
    "marginRight": "1.61rem",
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "blockquote *:last-child": Object {
    "marginBottom": 0,
  },
  "body": Object {
    "MozFontFeatureSettings": "\\"kern\\", \\"liga\\", \\"clig\\", \\"calt\\"",
    "WebkitFontFeatureSettings": "\\"kern\\", \\"liga\\", \\"clig\\", \\"calt\\"",
    "color": "hsla(0,0%,0%,0.8)",
    "fontFamily": "'Lato',sans-serif",
    "fontFeatureSettings": "\\"kern\\", \\"liga\\", \\"clig\\", \\"calt\\"",
    "fontKerning": "normal",
    "fontWeight": 400,
    "msFontFeatureSettings": "\\"kern\\", \\"liga\\", \\"clig\\", \\"calt\\"",
    "wordWrap": "break-word",
  },
  "code": Object {
    "fontSize": "0.85rem",
    "lineHeight": "1.61rem",
  },
  "dd": Object {
    "marginBottom": "1.61rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "dl": Object {
    "marginBottom": "1.61rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "dt": Object {
    "fontWeight": 700,
  },
  "fieldset": Object {
    "marginBottom": "1.61rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "figure": Object {
    "marginBottom": "1.61rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "form": Object {
    "marginBottom": "1.61rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "h1": Object {
    "color": "inherit",
    "fontFamily": "'Lato',sans-serif",
    "fontSize": "1.618rem",
    "fontWeight": 900,
    "lineHeight": 1.1,
    "marginBottom": "1.61rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
    "textRendering": "optimizeLegibility",
  },
  "h2": Object {
    "color": "inherit",
    "fontFamily": "'Lato',sans-serif",
    "fontSize": "1.33471rem",
    "fontWeight": 900,
    "lineHeight": 1.1,
    "marginBottom": "1.61rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
    "textRendering": "optimizeLegibility",
  },
  "h3": Object {
    "color": "inherit",
    "fontFamily": "'Lato',sans-serif",
    "fontSize": "1.21225rem",
    "fontWeight": 900,
    "lineHeight": 1.1,
    "marginBottom": "1.61rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
    "textRendering": "optimizeLegibility",
  },
  "h4": Object {
    "color": "inherit",
    "fontFamily": "'Lato',sans-serif",
    "fontSize": "1rem",
    "fontWeight": 900,
    "lineHeight": 1.1,
    "marginBottom": "1.61rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
    "textRendering": "optimizeLegibility",
  },
  "h5": Object {
    "color": "inherit",
    "fontFamily": "'Lato',sans-serif",
    "fontSize": "0.90825rem",
    "fontWeight": 900,
    "lineHeight": 1.1,
    "marginBottom": "1.61rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
    "textRendering": "optimizeLegibility",
  },
  "h6": Object {
    "color": "inherit",
    "fontFamily": "'Lato',sans-serif",
    "fontSize": "0.86558rem",
    "fontWeight": 900,
    "lineHeight": 1.1,
    "marginBottom": "1.61rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
    "textRendering": "optimizeLegibility",
  },
  "hgroup": Object {
    "marginBottom": "1.61rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "hr": Object {
    "background": "hsla(0,0%,0%,0.2)",
    "border": "none",
    "height": "1px",
    "marginBottom": "calc(1.61rem - 1px)",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "html": Object {
    "boxSizing": "border-box",
    "font": "112.5%/1.61 'Lato',sans-serif",
    "overflowY": "scroll",
  },
  "iframe": Object {
    "marginBottom": "1.61rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "img": Object {
    "marginBottom": "1.61rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "maxWidth": "100%",
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "kbd": Object {
    "fontSize": "0.85rem",
    "lineHeight": "1.61rem",
  },
  "li": Object {
    "marginBottom": "calc(1.61rem / 2)",
  },
  "li *:last-child": Object {
    "marginBottom": 0,
  },
  "li > ol": Object {
    "marginBottom": "calc(1.61rem / 2)",
    "marginLeft": "1.61rem",
    "marginTop": "calc(1.61rem / 2)",
  },
  "li > p": Object {
    "marginBottom": "calc(1.61rem / 2)",
  },
  "li > ul": Object {
    "marginBottom": "calc(1.61rem / 2)",
    "marginLeft": "1.61rem",
    "marginTop": "calc(1.61rem / 2)",
  },
  "noscript": Object {
    "marginBottom": "1.61rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "ol": Object {
    "listStyleImage": "none",
    "listStylePosition": "outside",
    "marginBottom": "1.61rem",
    "marginLeft": "1.61rem",
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "ol li": Object {
    "paddingLeft": 0,
  },
  "p": Object {
    "marginBottom": "1.61rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "p *:last-child": Object {
    "marginBottom": 0,
  },
  "pre": Object {
    "fontSize": "0.85rem",
    "lineHeight": "1.61rem",
    "marginBottom": "1.61rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "samp": Object {
    "fontSize": "0.85rem",
    "lineHeight": "1.61rem",
  },
  "strong": Object {
    "fontWeight": 700,
  },
  "table": Object {
    "borderCollapse": "collapse",
    "fontSize": "1rem",
    "lineHeight": "1.61rem",
    "marginBottom": "1.61rem",
    "marginLeft": 0,
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
    "width": "100%",
  },
  "td,th": Object {
    "MozFontFeatureSettings": "\\"tnum\\"",
    "WebkitFontFeatureSettings": "\\"tnum\\"",
    "borderBottom": "1px solid hsla(0,0%,0%,0.12)",
    "fontFeatureSettings": "\\"tnum\\"",
    "msFontFeatureSettings": "\\"tnum\\"",
    "paddingBottom": "calc(0.805rem - 1px)",
    "paddingLeft": "1.07333rem",
    "paddingRight": "1.07333rem",
    "paddingTop": "0.805rem",
    "textAlign": "left",
  },
  "th": Object {
    "fontWeight": 700,
  },
  "th:first-child,td:first-child": Object {
    "paddingLeft": 0,
  },
  "th:last-child,td:last-child": Object {
    "paddingRight": 0,
  },
  "thead": Object {
    "textAlign": "left",
  },
  "ul": Object {
    "listStyleImage": "none",
    "listStylePosition": "outside",
    "marginBottom": "1.61rem",
    "marginLeft": "1.61rem",
    "marginRight": 0,
    "marginTop": 0,
    "paddingBottom": 0,
    "paddingLeft": 0,
    "paddingRight": 0,
    "paddingTop": 0,
  },
  "ul li": Object {
    "paddingLeft": 0,
  },
}
`;
